Sources/OrderedCollections/OrderedSet/OrderedSet+SubSequence.swift

```diff
@@ -340,7 +340,7 @@ extension OrderedSet.SubSequence: RandomAccessCollection {
 extension OrderedSet.SubSequence: Equatable {
   @inlinable
   public static func ==(left: Self, right: Self) -> Bool {
-    left.elementsEqual(right)
+    left._base._elements == right._base._elements
```

Comment:
> This doesn't look right -- comparing _base._elements would compare the entire contents of the base sets of both inputs, rather than just the actual slices that they represent.

Ahh… that was a bad bug! Thanks!